pom.xml

```diff
@@ -56,12 +56,25 @@ under the License.
         <module>flink-shaded-asm-6</module>
         <module>flink-shaded-guava-18</module>
         <module>flink-shaded-netty-4</module>
+        <module>flink-shaded-netty-tcnative-dynamic</module>
         <module>flink-shaded-jackson-parent</module>
         <module>flink-shaded-hadoop-2</module>
         <module>flink-shaded-hadoop-2-uber</module>
     </modules>
 
     <profiles>
+        <profile>
+            <id>include-netty-tcnative-static</id>
```

Comment:
let's add a comment that this module is excluded by default since it is not compliant with the apache license. Bonus points for linking the INFRA ticked that you opened.